src/controllers/commands/CommandController.cpp

```diff
@@ -731,23 +731,42 @@ void CommandController::initialize(Settings &, Paths &paths)
         }
         if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
         {
+            auto gameName = words.mid(1).join(" ");
```

Comment:
I think this could be made const
```suggestion
            const auto gameName = words.mid(1).join(" ");
```